src/styles/view.styl

```diff
@@ -32,8 +32,12 @@
 
 .set-view-content-twofa
     &-modal
+      max-width 35rem !important
```

Comment:
We must not use `!important` (cc @GoOz), could you be more specific in your selector ?

Reply:
Done !